src/components/RegisterENS.vue

```diff
@@ -5,18 +5,20 @@
  */
 
 import { computed } from 'vue';
+import { useEns } from '@/composables/useEns';
+
+const { validEnsTlds } = useEns();
 
 const props = defineProps({
-  modelValue: String
+  modelValue: String,
+  validEnsTlds: Array
```

Comment:
This prop isn't needed, is it?

```suggestion
  modelValue: String
```